drivers/bt_ll_softdevice/CMakeLists.txt

```diff
@@ -4,5 +4,4 @@
 # SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
```

Comment:
The SDC flash driver is removed so the new one could be tested. I think it is possible to have both drivers present at the same time.